src/library_glemu.js

```diff
@@ -3650,7 +3650,7 @@ var LibraryGLEmulation = {
     GLImmediate.matricesModified = true;
     GLImmediate.matrixVersion[GLImmediate.currentMatrix] = (GLImmediate.matrixVersion[GLImmediate.currentMatrix] + 1)|0;
     GLImmediate.matrixStack[GLImmediate.currentMatrix].push(
-        Array.prototype.slice.call(GLImmediate.matrix[GLImmediate.currentMatrix]));
+        Array.from(GLImmediate.matrix[GLImmediate.currentMatrix]));
```

Comment:
`glPushMatrix` is in a hot path. `x.slice()` can be much faster than `Array.from()`, so this would be better with a 
```
GLImmediate.matrix[GLImmediate.currentMatrix].slice()
```

Reply:
Oh yes, we can do that in cases where we now we already have an array.

I had no idea that `Array.slice()` could be faster than `Array.from()`... do you know why?